st2actions/st2actions/utils/param_utils.py

```diff
@@ -108,14 +111,18 @@ def get_resolved_params(runnertype_parameter_info, action_parameter_info, action
 def _is_template(template_str):
     template_str = to_unicode(template_str)
     template = Template(template_str)
-    return template_str != template.render({})
+    try:
+        return template_str != template.render({})
+    except exceptions.UndefinedError:
+        return True
 
 
-def _renderable_context_param_split(action_parameters, runner_parameters):
+def _renderable_context_param_split(action_parameters, runner_parameters, base_context=None):
```

Comment:
Can base_context contain nested dictionaries? If it can, we should use copy.deepcopy.